APCAppCore/APCAppCore/UI/ViewControllers/APCPasscodeViewController.m

```diff
@@ -91,7 +93,7 @@ - (void)viewDidAppear:(BOOL)animated
     [super viewDidAppear:animated];
     
     if (![self.touchContext canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:nil]) {
-        [self.passcodeView becomeFirstResponder];
+        [self passcodeViewShouldBecomeFirstResponder];
```

Comment:
This naming is peculiar - normally a *Should* method would return a boolean and not take an action. Maybe `makePasscodeViewBecomeFirstResponder` or `passcodeViewBecomeFirstResponder`?